types/react-ranger/react-ranger-tests.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import { useRanger } from 'react-ranger';
+
+const App = () => {
+    const [values, setValues] = React.useState([10]);
+
+    const ranger = useRanger({
+        values,
+        onChange: setValues,
+        min: 0,
+        max: 100,
+        stepSize: 5,
+    });
+
+    return (
+        <>
+            <div
+                {...ranger.getTrackProps({
+                    style: {
+                        height: '4px',
+                        background: '#ddd',
+                        boxShadow: 'inset 0 1px 2px rgba(0,0,0,.6)',
+                        borderRadius: '2px',
+                    },
+                })}
+            >
+                {ranger.ticks.map(({ getTickProps }) => (
+                    <div
+                        {...getTickProps({
```

Comment:
It might be worth doing something like:

```tsx

const tickProps = getTickProps({className: 'someClass'}) // $ExpectType TickProps & { className: string }
```

Just to assert that the resultant type is what you expect since that's the only slightly complicated part of the types.

Reply:
I've added this to my test